cfpb/ccdb5-ui
# DATAP-1550 tileChartMap prevent state mutation

Update dependencies
Adding cloneDeep function to tileChartMap so it doesn't pollute redux state
to test:

yarn install
yarn start
toggle Map tab
confirm there is no warning that appears like so:

<img width="928" alt="Screenshot 2024-09-27 at 2 22 51 PM" src="https://github.com/user-attachments/assets/96311eae-075c-4760-ad87-27997e1ea497">
